heapster-saw/examples/iter_linked_list.sawcore

```diff
@@ -3,6 +3,22 @@ module iter_linked_list where
 
 import SpecM;
 
+-- Type description for the Mbox type, which is equivalent to Mbox_Ind
+ListDesc : TpDesc;
+ListDesc =
+    (Tp_Sum
+       Tp_Unit
+       (Tp_Pair
+          (varKindExpr Kind_Tp 1)
+          (varKindExpr Kind_Tp 0)));
+
+-- The type described by ListDesc
+ListTpF : TpDesc -> sort 0;
+ListTpF T = indElem (unfoldIndTpDesc (envConsElem Kind_Tp T nilTpEnv) ListDesc);
+
+primitive appendList : (T:TpDesc) -> ListTpF T -> ListTpF T -> ListTpF T;
+
+{-
```

Comment:
Should this chunk of commented-out code include a `FIXME` similar to the one in the commented-out `data Mbox` code in `mbox.sawcore`?

Reply:
Yep, I'll add that